collie/cross_validation.py

```diff
@@ -145,14 +146,16 @@ def stratified_split(interactions: BaseInteractions,
                      force_split: bool = False) -> Tuple[BaseInteractions, ...]:
     """
     Split an ``Interactions`` instance into train, validate, and test datasets in a stratified
-    manner such that each user appears at least once in each of the datasets.
+    manner such that each user or item appears at least once in each of the datasets. Whether
+    the split is on user of item is determined by the input ``Interactions`` instance.
 
-    This split guarantees that every user will be represented in the training, validation, and
-    testing datasets given they appear in ``interactions`` at least three times. If ``val_p ==
+    This split guarantees that every user or item will be represented in the training, validation,
```

Comment:
I think part of the `row/col` discussion was to make this consistent across all Collie files where we use `user or item`. What do you think about applying that to other files, like this one?

Reply:
I am kind of struggling with when to change to `row/col`. For this specific file, I think `user or item` makes more sense since we didn't change the `datasets.py` file and `Interactions` objects are still defined as rows = users and cols = items up to this point. The `row/col` switch only really happens in metrics. Happy to discuss this further.

Reply:
That makes total sense! Just for the sake of improved clarity, in these docstrings, can we say something along the lines of ```user or item, depending on ``negative_sample_type`` ``` just to make it clear when one applies. 

Maybe even something like: 

```
user or item, depending on ``negative_sample_type`` (user when ``negative_sample_type == 'item'``, item when user when ``negative_sample_type == 'user'``) 
```

although this might be overkill.